jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -326,4 +331,78 @@ private CaseMilestone createTestMilestone(String caseMilestoneId, String caseMil
 
         return milestone;
     }
+
+    private TaskSummary createTaskSummary(Long taskId, String name, String status) {
```

Comment:
Maybe it makes sense to rename the method to createTestTaskSummary, so that this method name is aligned with other helper methods in this class which serve the same purpose.